Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -301,6 +307,13 @@
 
 	private ScheduledFuture<?> appListingSchedule_;
 
+	/* BranchActivityLifeCycleObserver instance.Should be initialised on creating Instance with Application object. */
```

Comment:
Can you fix the spacing around the period here and everywhere else as we talked about?